apps/api/src/app/organization/organization.controller.ts

```diff
@@ -158,7 +182,12 @@ export class OrganizationController {
   }
 
   @Patch('/')
+  @ExternalApiAccessible()
   @Roles(MemberRoleEnum.ADMIN)
+  @ApiResponse(RenameOrganizationDto)
```

Comment:
Should be with 200?


```suggestion
  @ApiResponse(RenameOrganizationDto, 200)
```

Reply:
@ApiResponse() has 200 as default status code

Reply:
I see, good to know :)